backend/index.js

```diff
@@ -1,23 +1,29 @@
 const express = require("express");
 const mongoose = require("mongoose");
 const cors = require("cors");
-const routes = require("./routes");
 const dotenv = require("dotenv");
-const adminRouter = require("./routes/admin");
+const session = require("express-session");
+const passport = require("./middlewares/Passport");
+const routes = require("./routes");
+const authRoutes = require("./routes/authRoutes");
+const userRoutes = require("./routes/userRoutes");
 
-// Load environment variables from .env file
 dotenv.config();
 
 const app = express();
 
-// Middleware
 app.use(express.json());
 app.use(cors());
-// Routes
-app.use("/api", routes);
-app.use("/api/admin", adminRouter);
```

Comment:
ise hta diya kya ....admin routes

Reply:
Yes sir kyuki abhi change krna hoga 3 admin ke liye different api bnegi

Reply:
> ise hta diya kya ....admin routes

And in next pr I will try to add all admin routes accordingly to need